lib/kafo/hooking.rb

```diff
@@ -44,15 +45,15 @@ def loaded?
       @loaded
     end
 
-    def execute(group)
+    def execute(group, skip_stage_log: false)
```

Comment:
I would use the positive to avoid double negatives.
```suggestion
    def execute(group, log_stage: true)
```